rust/actyx/certs/src/bin/ax-cert.rs

```diff
@@ -32,13 +34,97 @@ struct AppLicenseOpts {
 
     /// ISO 8601 (i.e. 2014-11-28T12:00:09Z) expiration date time
     #[structopt(long)]
-    expires_at: DateTime<Utc>,
+    expires_at: Option<DateTime<Utc>>,
+
+    #[structopt(long, parse(try_from_str = parse_expires_in))]
+    expires_in: Option<DateTime<Utc>>,
 
     /// Requester's email address
     #[structopt(long)]
     email: String,
 }
 
+/// Parsing function for the `expires_in` variable.
+/// Accepts strings matching the regex: ^([0-9]+(Y|M|w|d|h|m|s))+$
```

Comment:
this isn’t true: it only finds one of each, and only in descending order or suffix magnitude

Reply:
The comment rot, nice catch